Dockerfile

```diff
@@ -10,7 +10,6 @@ COPY crates /build/crates
 COPY data /build/data
 COPY src /build/src
 COPY tests build/tests
-COPY diagnostic_templates /build/diagnostic_templates
```

Comment:
No longer needed as the default templates are now embedded into the app's binary.